examples/dft/run_time_dispatching/CMakeLists.txt

```diff
@@ -17,22 +17,31 @@
 # SPDX-License-Identifier: Apache-2.0
 #===============================================================================
 
-# NOTE: user needs to set env var SYCL_DEVICE_FILTER to use runtime example (no need to specify backend when building with CMake)
+# NOTE: user needs to set env var ONEAPI_DEVICE_SELECTOR to use runtime example (no need to specify backend when building with CMake)
 include(WarningsUtils)
 
 
 # Build object from all example sources
 set(DFT_RT_SOURCES "")
 # Set up for the right backend for run-time dispatching examples
 # If users build more than one backend (i.e. mklcpu and mklgpu, or mklcpu and CUDA), they may need to
-# overwrite SYCL_DEVICE_FILTER in their environment to run on the desired backend
+# overwrite ONEAPI_DEVICE_SELECTOR in their environment to run on the desired backend
 set(DEVICE_FILTERS "")
 if(ENABLE_MKLGPU_BACKEND OR ENABLE_CUFFT_BACKEND OR ENABLE_ROCFFT_BACKEND OR ENABLE_PORTFFT_BACKEND)
   list(APPEND DFT_RT_SOURCES "real_fwd_usm")
-  list(APPEND DEVICE_FILTERS "gpu")
 endif()
 
-message(STATUS "SYCL_DEVICE_FILTER will be set to the following value(s): [${DEVICE_FILTERS}] for run-time dispatching examples")
+if(ENABLE_MKLGPU_BACKEND OR ENABLE_PORTFFT_BACKEND)
```

Comment:
The portFFT backend can run on any device so it would be nice to use a different filter for it, maybe `*:gpu`. I verified this is accepted by `ONEAPI_DEVICE_SELECTOR`.
Technically the device type doesn't need to be a GPU either but we need to add something if we want to keep run the example with the current CMake logic.

Reply:
Done in [3d62cf9](https://github.com/oneapi-src/oneMKL/pull/484/commits/3d62cf917bdad28a06d4553fb75d48cd7deee27d). Testing was performed on a temporary branch and the only failures seen were already present in the base interfaces job. Thanks for the catch